app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/IncorporatedEntityInformation.scala

```diff
@@ -17,6 +17,7 @@
 package uk.gov.hmrc.incorporatedentityidentificationfrontend.models
 
 import play.api.libs.json._
+import uk.gov.hmrc.incorporatedentityidentificationfrontend.models.BusinessVerificationStatus.{businessVerificationPassKey, _}
```

Comment:
You may as well use ...BusinessVerificationStatus._

Reply:
I am going to fix it.